code/game/objects/items/storage/belt.dm

```diff
@@ -175,6 +175,15 @@
 		/obj/item/plunger
 		))
 
+/obj/item/storage/belt/medical/emt/PopulateContents()
+	new /obj/item/flashlight/pen(src)
+	new /obj/item/stack/medical/gauze/twelve(src)
+	new /obj/item/reagent_containers/syringe(src)
+	new /obj/item/reagent_containers/glass/bottle/epinephrine(src)

```

Comment:
epinephrine covers toxin damage as well.

Maybe the chem you'r looking for is calomel? In emergencies calomel is a great purger especially when the user is in crit.

Pent seems a bit much imo.

Reply:
Unless the wiki isn't updated epi is for stopping people that are crit from dying, isn't it? I think the idea behind the penetic is to remove radiation OR purge bad chems.

Reply:
Yeah, I'm not up to speed with cobby chems tbh. My intention was giving EMTs 1) something that keeps people in crit from dying immediately, 2) something to preserve fresh corpses and 3) something that halts tox damage until they get to medbay

Reply:
epi has always covered tox/brute/burn evil wiki readers.